js/scripts/util.js

```diff
@@ -61,7 +61,15 @@ const SFX = {
     SLINGSHOT_LAUNCH3: {path: "./sfx/launch3.mp3", volume: 0.5},
     SLINGSHOT_LAUNCH4: {path: "./sfx/launch4.mp3", volume: 0.5},
     SLINGSHOT_STRETCH: {path: "./sfx/slingshot_stretch.mp3", volume: 0.4},
-    SONIC_DASH: {path: "./sfx/sonic_dash.mp3", volume: 0.2}
+    SWORD_SWING1: {path: "./sfx/sword_swing1.mp3", volume: 0.2},
```

Comment:
It might be fairly easy to add a volume slider for sfx when the game is paused.

Reply:
That's what I'm thinking too. There's a method in AssetManager that could be used to modify global volume.